core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/Purchases.ios.kt

```diff
@@ -16,23 +19,21 @@ import com.revenuecat.purchases.kmp.models.StoreTransaction
 import com.revenuecat.purchases.kmp.models.SubscriptionOption
 import com.revenuecat.purchases.kmp.strings.ConfigureStrings
 import platform.Foundation.NSURL
-import cocoapods.PurchasesHybridCommon.RCDangerousSettings as IosDangerousSettings
-import cocoapods.PurchasesHybridCommon.RCPurchases as IosPurchases
 
 public actual class Purchases private constructor(private val iosPurchases: IosPurchases) {
```

Comment:
There are a bunch of code style changes in this file. We don't really have one setup, so for now we're using the Android Studio default. Do you have it set to something custom?

Reply:
I'm honestly not sure how these got added in - I'm using the most vanilla installation of Android Studio imaginable 😅 Maybe I opened the file in VSCode and that reformatted it? Either way, I'll see if I can revert the styling changes.

Reply:
Okay, I think I got them all reverted! 😄

Reply:
VSCode might have done it indeed 🤔 

You can auto-format using CMD+OPTION+L. You were nearly there haha. Did the rest here: https://github.com/RevenueCat/purchases-kmp/pull/149.

Reply:
Merged!